.circleci/config.yml

```diff
@@ -157,6 +157,9 @@ jobs:
       - with_cache:
           cache_version: << parameters.clojure_version >>|<< parameters.jdk_version >>
           steps:
+            - run:
```

Comment:
I'm a bit puzzled what does this replace. I see it added, but I don't see anything changed/removed.

Reply:
Ah, I finally understand what you mean. 

Might be good to add some comment here explaining all of this.